plugins/azure/virtualnetworks/noVPNgateways.js

```diff
@@ -0,0 +1,67 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'No VPN Gateways',
+    category: 'Virtual Networks',
+    domain: 'Network Access Control',
+    description: 'Ensures that virtual network gateways type is not VPC.',
+    more_info: 'Use ExpressRoute as gateway type instead of VPN for virtual network gateways to meet your organization\'s security and compliance requirements.',
+    link: 'https://learn.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-about-vpngateways',
+    recommended_action: 'Ensure gateway type is not configured as VPC for all virtual network gateways.',
```

Comment:
```suggestion
    recommended_action: 'Ensure gateway type is not configured as VPN for all virtual network gateways.',
```
maybe it should be something like 'Delete all VPN gateways for your virtual networks'